lib/menu.rb

```diff
@@ -0,0 +1,19 @@
+# deals with menus and selection
+
+class Menu
+
+	attr_reader :dishes, :prices
+
+	def initialize
+		@dishes = Hash.new
```

Comment:
Style/EmptyLiteral: Use hash literal `{}` instead of `Hash.new`. (https://github.com/bbatsov/ruby-style-guide#literal-array-hash)